docs/users/faq.md

```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
 are available. In addition, plugins that support BungeeCord but only require
 that they are installed on the server (nothing on the proxy) typically use
 the BungeeCord plugin messaging channel, which can be supported by BungeeQuack
-for Velocity 1.0.x or natively by Velocity 1.1.0.
\ No newline at end of file
+for Velocity 1.0.x or natively by Velocity 1.1.0.
+
+# Common Errors
+
+``
```

Comment:
Let's add some headers?

Reply:
I have added some headers in the most recent commit. Let me know if that is what you were looking for.